### PR TITLE
Add dita.xsl.html5-bootstrap extension point.

### PR DESCRIPTION
**Pull request by jason-fox**
When creating Bootstrap plugins, the relevant XSLT should only fire if the `html5-bootstrap` transform is run. Therefore a new extension point is required to stop these dependent plug-ins from polluting the standard HTML5 transform instead.

It is necessary to add a template file and duplicate the import of standard HTML5, then add standard `dita.xsl.html5` overrrides, then import the standard bootstrap elements before finally adding  the new `dita.xsl.html5-bootstrap` extension point.

The standard HTML5 imports were last updated six years ago, if this plugin still wishes to support DITA-OT 3.0, 3.1 and 3.2 it needs to fallback to an older list of HTML5 imports without hazard statements  to ensure that the generated XSLT still compiles.

### CODE REVIEW

File: xsl/dita2html5-bootstrap-3.0.xsl [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright © 2017 · infotexture · Roger W. Fienhold Sheen -->
+<!-- See the accompanying LICENSE file for applicable license -->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="3.0">
+
+  <!--
+    For DITA-OT 3.0, 3.1 and 3.2 import a version of the
+    HTML transform without reference to Hazard Statements
+  -->
```
Comment (infotexture) [inline xsl/dita2html5-bootstrap-3.0.xsl:9]: @jason-fox Apart from this comment, this file is functionally identical to `dita2html5-bootstrap.xsl`. 

Is that intentional?

File: Customization/xsl/tables.xsl
Comment (infotexture) [inline Customization/xsl/tables.xsl:1]: The changes in `tables.xsl` don't seem directly related to the new extension point that this PR adds.

Are they intended for inclusion in a separate PR, or do we need them here?

**Review by infotexture — COMMENTED** [review]
Makes sense to add this new extension point, but not sure about a few details (see line comments).

> if this plugin still wishes to support DITA-OT 3.0, 3.1 and 3.2

I think that still makes sense (at least for the 5.3.x versions).

I'd be fine requiring DITA-OT 4.x sometime soon, but maybe that should wait for Bootstrap 5.4 (or whatever's next there).

File: Customization/xsl/tables.xsl
Comment (jason-fox) [reply Customization/xsl/tables.xsl:1]: Probably should have been included in  one of the other minor CSS improvement  PRs, but doesn't do any harm adding it here:

-  `ps-3 pe-3` means a padding of 3 at start  of an element and a padding of 3 at end  of an element 
- `px-3` means a padding of 3 at start and end  of an element 

Functionally they are both the same,   `px-3` is shorter.

File: xsl/dita2html5-bootstrap-3.0.xsl (hunk 1 above)
Comment (jason-fox) [reply xsl/dita2html5-bootstrap-3.0.xsl:9]: No. Legacy should only be included in the 3.0 version. Fixed - 8cf53e6

**Pull request merged**